plugins/alerts/frontend/public/templates/vue-main.html

```diff
@@ -0,0 +1,295 @@
+<script type="text/x-template" id="alerts-home">
+    <div id="alerts-view" v-bind:class="[componentId]">
+        <cly-header>
+            <template v-slot:header-left>
+                <h2> {{i18n('alert.plugin-title')}}
+                </h2>
+                <span class="ion-help-circled color-cool-gray-50" style="margin-left:10px;" v-tooltip.top-center="i18n('alert.tips')">
+                </span>
+            </template>
+            <template v-slot:header-right>
+                <div id="create-alert" v-if="canCreate" class="cly-vue-button button-green-skin" v-on:click="createAlert()"><i class="el-icon-circle-plus"/> {{i18n('alert.Add_New_Alert')}}</div>
```

Comment:
please remove the **cly-vue-button button-green-skin** classes.
instead refer to the recent docs shared regarding core components in cly-vue channel.
you should use
<el-button type="success"></el-button>